text.md

```diff
@@ -0,0 +1 @@
+<p>This file should be ignored by git</p>
```

Comment:
2- It's a project requirement to add a `test.md` file in the `.gitignore file`. Accidentally, you misspelled your file name to `text.md` rather than `test.md`. Consider changing it and ensuring that both `test.md` file and the `node_modules` directory are not present on GitHub as they need to be ignored with help of the `.gitignore file`.      

![image](https://user-images.githubusercontent.com/94223367/218786424-1a2b1780-4cc7-4776-add3-208bcd00701b.png)